apps/docs/src/app/core/component-docs/info-label/info-label-docs.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component } from '@angular/core';
+import { Schema } from '../../../schema/models/schema.model';
+import { SchemaFactoryService } from '../../../schema/services/schema-factory/schema-factory.service';
+import * as InfoLableDefaultExample from '!raw-loader!./examples/info-label-default-example.component.html';
+import * as InfoLableTextExample from '!raw-loader!./examples/info-label-text-example.component.html';
+import * as InfoLabelIconTextExample from '!raw-loader!./examples/info-label-text-icon-example.component.html';
+import * as InfoLableNumericIconExample from '!raw-loader!./examples/info-label-icon-numeric-example.component.html';
+import { ExampleFile } from '../../../documentation/core-helpers/code-example/example-file';
+import { Icons } from '../../../documentation/utilities/icons';
+
+@Component({
+    selector: 'app-info-label',
+    templateUrl: './info-label-docs.component.html'
+})
+export class InfoLabelDocsComponent {
+
+    defaultInfoLabelHtmlType: ExampleFile[] = [
+        {
+            language: 'html',
+            code: InfoLableDefaultExample,
+            fileName: 'Info-label-default-example',
```

Comment:
I think it should be:
`fileName: 'Info-label-default-example',` => `fileName: 'info-label-default-example',`